### ISSUE TITLE
Correct major flaw that Requests are connected to Roles instead of Institutions

### ISSUE THREAD

**Issue by cannedit** [DiSSCo/ELViS#236]
#### Description
NHM Evaluation Excel sheet no. 8:
Role conflict to be resolved / roles structure:  It should be made possible to combine some roles (e.g. requester & TAF administrator) + other role related issues.

#### Agreement
The very first thing we have to do when we start developing again is to look closely at the roles system, because shortly before the definitive deadline of ELViS Development Phase 1 we discovered some conceptual flaws in there, which we couldn't solve anymore, hence we then asked you to be very careful with combining and switching roles during the call.
Basically it comes down to Requests being connected to roles/persons rather than to Institutions which causes Requests to disappear when persons switch roles/institutions. We will correct this.

**Comment by cannedit** [comment]
Please test this a.s.a.p., because it's quite important that we know this bug has been fixed properly.

**Comment by mmarschler** [comment]
I tested the connection/visability of TA requests with my 3 profiles, it seems to work well!

**as "normal" requester** (maria@marschler.at): I can only see my own requests (in "all requests" and "my own requests")

**as TAF admin for Belgium** (synthesys@nhm-wien.ac.at): I can only see TA requests for Belgium (RMCA & Meise; in "all requests" and "requests assigned to me")
This profile is also institutional moderator and VA coordinator from NHMW, so there seems to be no role conflict concerning new TA requests - it works well!

**as TA coordinator for NHMW** (maria.marschler@nhm-wien.ac.at): I can only see the requests for NHMW (in "all requests" and "requests assigned to me")
This profile is also TA scorer for Austria. So it was not possible to for me to test these roles separately.
